### PR TITLE
Feat/mixed changes

### PR DESCRIPTION
**Pull request by ccdd13**
### git log
<pre>* <font color="#AA5500">434675d (</font><font color="#FF5555"><b>origin/feat/mixed-changes</b></font><font color="#AA5500">)</font> no guest user by default
* <font color="#AA5500">bdc8249</font> put $auth_users &amp; $readonly_users together
* <font color="#AA5500">9caee73</font> added .gitignore
* <font color="#AA5500">3330a76</font> replace doc url for $datetime_format from function.date.php to datetime.format.php
* <font color="#AA5500">fa2b846</font> safe include config.php with __DIR__
* <font color="#AA5500">4e163af</font> remove trailing whitespace
* <font color="#AA5500">a19ff20 (</font><font color="#FF5555"><b>upstream/master</b></font><font color="#AA5500">)</font> Update vendor CDN versions
</pre>

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -21,19 +21,16 @@
 // Generate secure password hash - https://tinyfilemanager.github.io/docs/pwd.html
 $auth_users = array(
     'admin' => '$2y$10$/K.hjNr84lLNDt8fTXjoI.DBp6PpeyoJ.mGwrrLuCZfAwfSAGqhOW', //admin@123
-    'user' => '$2y$10$Fg6Dz8oH9fPoZ2jJan5tZuv6Z4Kp7avtQ9bDfrdRntXtPeiMAZyGO' //12345
 );
```
Comment (prasathmani) [inline config.php:25]: add read-only user option by default

**Comment by ccdd13** [comment]
### git log
<pre>* <font color="#AA5500">6a37c37 (</font><font color="#55FFFF"><b>HEAD -&gt; </b></font><font color="#55FF55"><b>feat/mixed-changes</b></font><font color="#AA5500">, </font><font color="#FF5555"><b>origin/feat/mixed-changes</b></font><font color="#AA5500">)</font> update $datetime_format
* <font color="#AA5500">304f257</font> put $auth_users &amp; $readonly_users together
* <font color="#AA5500">3330a76</font> replace doc url for $datetime_format from function.date.php to datetime.format.php
* <font color="#AA5500">fa2b846</font> safe include config.php with __DIR__
* <font color="#AA5500">4e163af</font> remove trailing whitespace
* <font color="#AA5500">a19ff20 (</font><font color="#FF5555"><b>upstream/master</b></font><font color="#AA5500">)</font> Update vendor CDN versions
</pre>

File: config.php [hunk 2]
```diff
@@ -67,8 +67,8 @@
 $iconv_input_encoding = 'UTF-8';
 
 // date() format for file modification date
-// Doc - https://www.php.net/manual/en/function.date.php
-$datetime_format = 'd.m.y H:i';
+// Doc - https://www.php.net/manual/en/datetime.format.php
+$datetime_format = 'd.m.y H:i:s';
```
Comment (ccdd13) [inline config.php:71]: added _second_ to file modification date format

Comment (ccdd13) [reply config.php:71]: my preference for _modification date_
but i can remove it with `git rebase -i` , if you don't prefer.

**Review by prasathmani — APPROVED** [review]

**Pull request merged**